RELEASE_NOTES.md

```diff
@@ -28,16 +28,7 @@ making a backup is strongly advised.
 
 ## The Big Ones
 
-- A full rewrite of the lighttable has been undertaken. This gives again a
-  big performance improvement and will allow smooth interaction at any
-  screen resolution. At the same time the filmstrip has been reworked.
-
-  It is important to note that this speed improvement is really
-  noticeable, particularly in the zoomable lighttable view.
-
-  The culling view has also been rewritten from scratch.
```

Comment:
I'm ok with all changes except this. I think we should keep the culling view and 8k notes. Can you propose an update with to this PR? TIA.

Reply:
@TurboGit yes, done and pushed.